TouchBar.h

```diff
@@ -81,4 +81,16 @@ extern void DFRSystemModalShowsCloseBoxWhenFrontMost(BOOL flag) ;
 
 @end
 
+// MARK: NSFunctionRow
+@interface NSFunctionRow: NSObject
++ (void)markActiveFunctionRowsAsDimmed:(BOOL)arg1;
++ (nonnull NSArray *)_topLevelFunctionRowViews;
++ (nonnull NSArray *)activeFunctionRows;
++ (void)removeActiveFunctionRow:(nonnull NSFunctionRow *)arg1;
++ (void)addActiveFunctionRow:(nonnull NSFunctionRow *)arg1;
+
```

Comment:
Are you including this because it's a private class and we don't have a header for it? if a simple `@class NSFunctionRow` isn't sufficient then, end it here because our new methods aren't *in* the original class.
~~~
@end
~~~

These are defined in our category, not in the original class
~~~
@interface NSFunctionRow (interceptor)
~~~